### PR TITLE
To do add remove

### PR DESCRIPTION
**Pull request by ntzwilly**
# To Do list: add & remove
## Project requirements
- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for editing task descriptions.
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for the "Clear all completed" button (use filter() method).
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length.
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- All changes to the To-Do List should be saved in local storage.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      todoTasks.push({
+        description: newToDo,
+        completed: false,
+        index: todoTasks.length,
+        checked: false,
+      });
+      savedList();
+    }
+  });
+}
```
Comment (DeliceLydia) [inline src/crud.js]: - [x]  The property `index` has to be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). please fix that!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @ntzwilly , 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 1 above)
Comment (ntzwilly) [reply src/crud.js]: Thank you, I have made the requested modification

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ntzwilly 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**